packages/ui-flex/src/Flex/index.tsx

```diff
@@ -47,31 +48,26 @@ import type { FlexProps } from './props'
 category: components
 ---
 @module Flex
+@tsProps
 **/
 @withStyle(generateStyle, generateComponentTheme)
 class Flex extends Component<FlexProps> {
   static readonly componentId = 'Flex'
 
-  constructor(props: FlexProps) {
-    super(props)
-    // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
-    props.makeStyles()
```

Comment:
It was probably left like this when we did the themeable migration, so I refactored it.